### ISSUE TITLE
node-windows : package dependencies from Git dosen't work with nanoserver 2016/1709/1803

### ISSUE THREAD

**Issue by stalb** [StefanScherer/dockerfiles-windows#372]
I can't install dependencies from git repo with node-windows docker images (nanoserver-2016/1709/1803).

How to reproduce the pb: start node-windows container and install module from git. 
It works with nanoserver-1809, but fails using nanoserver-2016, nanoserver-1709 and nanoserver-1803.

`docker run -it --rm  stefanscherer/node-windows:10.14.1-nanoserver-1803 cmd`
```
Microsoft Windows [Version 10.0.17134.407]
(c) 2018 Microsoft Corporation. All rights reserved.

C:\>npm install git+https://github.com/kelektiv/node-uuid.git#v3.3.2
npm info it worked if it ends with ok
npm info using npm@6.4.1
npm info using node@v10.14.1
npm timing npm Completed in 271ms
npm info ok
npm info it worked if it ends with ok
npm info using npm@6.4.1
npm info using node@v10.14.1
npm timing stage:rollbackFailedOptional Completed in 0ms
npm timing stage:runTopLevelLifecycles Completed in 4098ms
npm ERR! code 57
npm ERR! Command failed: C:\git\cmd\git.EXE submodule update -q --init --recursive
npm ERR!
npm timing npm Completed in 6600ms

npm ERR! A complete log of this run can be found in:
npm ERR!     C:\Users\ContainerAdministrator\AppData\Roaming\npm-cache\_logs\2018-12-20T01_17_40_302Z-debug.log
```
I thought first that it was because git wasn't installed, but discovered that it was since #336 .

I'm using windows 10 version 1809
Docker version: 
```
Client: Docker Engine - Community
 Version:           18.09.0
 API version:       1.39
 Go version:        go1.10.4
 Git commit:        4d60db4
 Built:             Wed Nov  7 00:47:51 2018
 OS/Arch:           windows/amd64
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          18.09.0
  API version:      1.39 (minimum version 1.24)
  Go version:       go1.10.4
  Git commit:       4d60db4
  Built:            Wed Nov  7 00:56:41 2018
  OS/Arch:          windows/amd64
  Experimental:     true
```
Docker info:
```
Containers: 6
 Running: 0
 Paused: 0
 Stopped: 6
Images: 192
Server Version: 18.09.0
Storage Driver: windowsfilter (windows) lcow (linux)
 Windows:
 LCOW:
Logging Driver: json-file
Plugins:
 Volume: local
 Network: ics l2bridge l2tunnel nat null overlay transparent
 Log: awslogs etwlogs fluentd gelf json-file local logentries splunk syslog
Swarm: inactive
Default Isolation: hyperv
Kernel Version: 10.0 17763 (17763.1.amd64fre.rs5_release.180914-1434)
Operating System: Windows 10 Pro Version 1809 (OS Build 17763.194)
OSType: windows
Architecture: x86_64
CPUs: 8
Total Memory: 7.891GiB
Name: ASUS_ST
ID: P2M6:HIL5:MKMA:OJRJ:CEU7:OWT6:UXWF:DEVZ:ZYTE:CTDE:CWQV:W4I6
Docker Root Dir: C:\ProgramData\Docker
Debug Mode (client): false
Debug Mode (server): true
 File Descriptors: -1
 Goroutines: 30
 System Time: 2018-12-19T19:09:45.7729302+01:00
 EventsListeners: 1
Registry: https://index.docker.io/v1/
Labels:
Experimental: true
Insecure Registries:
 127.0.0.0/8
Live Restore Enabled: false
Product License: Community Engine
```

**Comment by stalb** [comment]
It's working if I replace MinGit with MinGit-busybox in the docker images

**Comment by StefanScherer** [comment]
@stalb If you want to send me a PR with the fix I would like to merge it.

**Comment by StefanScherer** [comment]
Fixed with #375
